Core/ModuleInstaller.cs

```diff
@@ -452,10 +452,17 @@ internal static List<InstallableFile> FindInstallableFiles(ModuleInstallDescript
                     installDir = ksp == null ? null : ksp.Tutorial();
                     makeDirs = true;
                     break;
+
+                case "Scenarios":
```

Comment:
@Postremus : I'm not sure that would reduce our complexity at all (one has makeDirs false, two have makrDirs true, all install to different locations). But I'd welcome a commit that simplifies things if you have one! (I made sure to do this in the KSP-CKAN repo for easy adjustments)